frontend/src/pages/AdminReviewProposalEarmark.svelte

```diff
@@ -100,7 +100,7 @@
   {#if proposal }
     <Section
             title={proposal.proposalTitle}
-            description={proposal.proposalDescription}
+            description={proposal.proposalDescription + '\n\n' + proposal.discourseLink}
```

Comment:
Does this not have to be written out as MD?

I was thinking that perhaps this could be formatted a bit better.... and perhaps, by making the title into an href, rather than adding a bit url to the end of the description.... 

Just providing feedback.
This works for now, we can improve later.

Approved!

Reply:
Ugh this might be in HTML I'm not sure if it's markdown. I should replace `\n` with `<br/>`. Nice catch!!